bemserver_ui/templates/pages/services/cleanup/manage.html

```diff
@@ -33,21 +33,37 @@ <h5>Service state</h5>
         </div>
     </div>
     <div class="row">
-        <div class="col">
+        <div class="col"> 
+            <div class="d-flex justify-content-between align-items-center">
+                <div class="d-flex align-items-center gap-2">
+                    <form id="formFiltersElmt" class="d-lg-flex d-grid gap-2" action="{{ url_for(request.endpoint) }}" method="POST">
+                        <input type="hidden" name="sort" id="sort" value="{{ sort }}">
+                        <label class="form-label text-nowrap text-muted mb-0" for="sort">Sort by</label>
+                        <input type="radio" class="btn-check" name="sort" id="sort-timeseries_name-asc" autocomplete="off" {% if sort == "+timeseries_name" %} checked{% endif %}>
+                        <label class="btn btn-sm btn-outline-info" for="sort-timeseries_name-asc" title="Name alphabetical order"><i class="bi bi-sort-alpha-down"></i> Name</label>
+                        <input type="radio" class="btn-check" name="sort" id="sort-timeseries_name-desc" autocomplete="off" {% if sort == "-timeseries_name" %} checked{% endif %}>
+                        <label class="btn btn-sm btn-outline-info" for="sort-timeseries_name-desc" title="Name reverse alphabetical order"><i class="bi bi-sort-alpha-up"></i> Name</label>
+                        <input type="radio" class="btn-check" name="sort" id="sort-last_timestamp-asc" autocomplete="off"{% if sort == "+last_timestamp" %} checked{% endif %}>
+                        <label class="btn btn-sm btn-outline-info" for="sort-last_timestamp-asc" title="Timestamp date order"><i class="bi bi-sort-numeric-down"></i> Timestamp</label>
+                        <input type="radio" class="btn-check" name="sort" id="sort-last_timestamp-desc" autocomplete="off"{% if sort == "-last_timestamp" %} checked{% endif %}>
+                        <label class="btn btn-sm btn-outline-info" for="sort-last_timestamp-desc" title="Timestamp reverse date order"><i class="bi bi-sort-numeric-up"></i> Timestamp</label>
+                    </form>
+                </div>
+                <small class="text-nowrap text-muted">
+                    <app-items-count first-item="1" last-item="{{ cleanup_timeseries | length }}" total-count="{{ cleanup_timeseries | length }}"></app-items-count>
+                </small>
+            </div>
             <div class="table-responsive-xl">
```

Comment:
Close here the previous div row and open a new one.
Add a margin bottom (mb-2 for example) on the row before the table, to have a little space between sort buttons and table.